src/DebugConfigurationProvider.spec.ts

```diff
@@ -53,7 +53,8 @@ describe('BrightScriptConfigurationProvider', () => {
             activeDeviceManager,
             null,
             vscode.window.createOutputChannel('Extension'),
-            userInputManager
+            userInputManager,
+            null // BrightScriptCommands is not used in this test (help!)
```

Comment:
```suggestion
            null // BrightScriptCommands is not used in this test
```

`null` is fine here. No need to do more work than the test requires.